resources/views/auth/event-owners/sign-in.blade.php

```diff
@@ -2,8 +2,8 @@
 
 @section('content')
 <div class="p-0 overflow-auto" style="width: 100vw; height: 100vh;">
-    <div class="d-flex justify-content-center align-items-stretch w-100 h-100" style="width: 100vw; height: 100vh;">
-        <div class="w-50 p-0 d-flex flex-column justify-content-center">
+    <div class="row d-flex justify-content-center align-items-stretch w-100 h-100" style="width: 100vw; height: 100vh;">
```

Comment:
Please refrain from using in-line css
Create another class for the div and add the styles there